src/MongoDB/BulkWrite.c

```diff
@@ -242,12 +242,8 @@ static bool php_phongo_bulkwrite_update_apply_options(bson_t* boptions, zval* zo
 	bool multi = false, upsert = false;
 
 	if (zoptions) {
-		if (php_array_existsc(zoptions, "multi")) {
-			multi = php_array_fetchc_bool(zoptions, "multi");
-		}
-		if (php_array_existsc(zoptions, "upsert")) {
-			upsert = php_array_fetchc_bool(zoptions, "upsert");
-		}
+		multi = php_array_fetchc_bool(zoptions, "multi");
+		upsert = php_array_fetchc_bool(zoptions, "upsert");
```

Comment:
[PHPC-1073](https://jira.mongodb.org/browse/PHPC-1073) mentions that bool fetching might still want "exists" checks, but I assume this is fine since both default to `false` already, correct?

Likewise for any bool fetches later on in this PR where `false` corresponds to the default case.